styles/style.css

```diff
@@ -80,6 +80,87 @@ ul li {
   display: none;
 }
 
+/* Modal menu styles */
+.modal-container {
+  /* Position */
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 0%;
+  height: 100%;
+  z-index: 1;
+
+  /* Theme */
+  background: #6070ff;
+  mix-blend-mode: multiply;
+  transition: width 0.3s linear;
```

Comment:
- [ ] Good job implementing a good working menu modal :clap: 
, Kindly make sure the modal's design is the same as the Figma, by  margins and paddings, and also as you can see below the background is __blur__:

Your design:
![notBlur2](https://user-images.githubusercontent.com/97834160/209232424-08ecc69e-ecbf-4de1-883f-508d6d6cebae.png)
Figma design:

![blur](https://user-images.githubusercontent.com/97834160/209232456-e2e8dd23-490b-4350-9f32-6327b1aa1c76.png)

Reply:
Hi @GedeonTS
Thanks for your nice review, however, I changed the `background-color` blur issue, but I can see there is no `padding` or `margin` issue, please have a look.
Thanks:)